.circleci/config.yml

```diff
@@ -161,49 +203,79 @@ jobs:
       - store_artifacts:
           path: test-results
 
-  unpackaged-sanity:
+
+  linux-tarball-test:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/repo
+
+      # We'll want to use the LTS version of node
+      - update_node:
+          os: linux
+
+      # Set up the plugin
+      - run:
+          name: "Install SFDX and tarball"
+          command: |
+            npm install -g sfdx-cli
+            # We need to determine the name of the tarball
+            SFDC_TARBALL=$(ls | grep salesforce*.tgz)
+            # Pipe in a 'y' to simulate agreeing to install an unsigned plugin. Use a URI of the file's full path.
+            echo y | sfdx plugins:install file://`pwd`/${SFDC_TARBALL}
+
+      # Run through the smoke test script. If everything passes, we can assume that the plugin is stable.
+      - run:
+          name: "smoke test"
+          command: smoke-tests/smoke-test.sh sfdx
+
+      # Since the results are expected to be non-null, upload them as an artifact to avoid tanking the tests.
+      - store_artifacts:
+          path: smoke-test-results
+
+  windows-tarball-test:
     executor:
       name: win/default # executor type
       size: "medium"
     working_directory: C:\repo
-
     steps:
-      # Commands are run in a Windows virtual machine environment
+      # Commands are run in a Windows VM environment
       - attach_workspace:
           at: C:\repo
 
+      # Update node to LTS
+      - update_node:
+          os: windows
+
+      # Set up the plugin
```

Comment:
This block to install the plugin is currently duplicated on windows and linux, can it be refacctored into a reusable command?

Reply:
Probably. I'll provide an update if it turns out I can't for some reason.

Reply:
@jfeingold35 - can this be refactored?

Reply:
I think this just slipped my mind with the last commit. Oops.